src/main/java/com/ays/assignment/model/dto/request/AssignmentLocationRequest.java

```diff
@@ -0,0 +1,12 @@
+package com.ays.assignment.model.dto.request;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class AssignmentLocationRequest {
```

Comment:
Eğer bu sınıf en uygun atamayı bulmak için kullanılacak ise `AssignmentSearchRequest` olarak adlandırmamız daha doğru olur

Reply:
Gerekli değişiklikleri yaptım. Kontrol edebilir misin? @agitrubard